### ISSUE TITLE
Unable to create wildcard certificates for DuckDNS domains until after running certbot manually.

### ISSUE THREAD

**Issue by travipross** [jc21/nginx-proxy-manager#1156]
**Checklist**
- Have you pulled and found the error with `jc21/nginx-proxy-manager:latest` docker image?
  - Yes
- Are you sure you're not using someone else's docker image?
  - Yes
- Have you searched for similar issues (both open and closed)?
  - Yes

**Describe the bug**
Unable to create a wildcard SSL certificate for a DuckDNS domain, using the DNS challenge with the DuckDNS provider selected. Error states: "The clearing of the TXT record for domain was not successful". 

Able to `exec` into the NPM container and run `certbot` manually using the same DuckDNS token, with success. Subsequently trying to create the same SSL certificate from the NPM UI is successful. 

**Nginx Proxy Manager Version**
v2.9.3

**To Reproduce**
1. Go to SSL certificates
2. Click on Add SSL certificate > Let's Encrypt
3. Enter a valid DuckDNS domain name with a wildcard subdomain (`*.<yourdomain>.duckdns.org`). 
4. Enable "Use a DNS Challenge" 
5. Select DuckDNS from the DNS Provider dropdown
6. Populate your private token for the Credentials File Content input (`dns_duckdns_token=your-duckdns-token`) 
7. Set propagation seconds to 60
8. Agree to ToS and click Save. Wait for certbot error to appear in UI.
9. `docker exec -it <your-npm-container-name> bash`
10. Run the following: 
```
certbot certonly \
  --non-interactive \
  --agree-tos \
  --email my@email.com \
  --preferred-challenges dns \
  --authenticator dns-duckdns \
  --dns-duckdns-token <your-duckdns-token> \
  --dns-duckdns-propagation-seconds 60 \
  -d "*.<yourdomain>.duckdns.org"
```
11. Cert should be successfully generated, with files stored in `/etc/letsencrypt` directory.
12. Repeat steps 1-8 above from NPM UI. Wildcard certificate is successfully created within NPM.

**Expected behavior**
Wildcard certificates should be created from the beginning

**Screenshots**

**Operating System**
Ubuntu 20.04 LTS

**Additional context**
Docker version 20.10.7, build f0df350

**Comment by chaptergy** [comment]
The error message shown should also include the certbot command executed if I remember correctly. Could you test that exact command to see if that works? Maybe even post the entire error message. Does this only happen for wildcard certificates or for any certificate?

**Comment by travipross** [comment]
Sure, so the exact error output when running from the UI is this (sanitized for my own personal info):
```
Error: Command failed: /opt/certbot/bin/certbot certonly --non-interactive --cert-name "npm-45" --agree-tos --email "<my-email>" --domains "*.<my-domain>.duckdns.org" --authenticator dns-duckdns  --dns-duckdns-propagation-seconds 60 --dns-duckdns-token dns_duckdns_token=<my-token>
Saving debug log to /var/log/letsencrypt/letsencrypt.log
Plugins selected: Authenticator dns-duckdns, Installer None
Performing the following challenges:
dns-01 challenge for <my-domain>.duckdns.org
Cleaning up challenges
Encountered exception during recovery: certbot.errors.PluginError: The clearing of the TXT record for domain "<my-domain>.duckdns.org" was not successful.
Request status code: 200
Request response text: KO
The TXT update "muJ13qsxtlfcYOVZqsW-7pLnFfVe131LusTIgkX_cx5" for domain "<my-domain>.duckdns.org" could not be set.
Request status code: 200
Request response text: KO

    at ChildProcess.exithandler (node:child_process:326:12)
    at ChildProcess.emit (node:events:369:20)
    at maybeClose (node:internal/child_process:1067:16)
    at Process.ChildProcess._handle.onexit (node:internal/child_process:301:5)
```
If I run the failed command manually in the container (`/opt/certbot/bin/certbot certonly --non-interactive --cert-name "npm-45" --agree-tos --email "<my-email>" --domains "*.<my-domain>.duckdns.org" --authenticator dns-duckdns  --dns-duckdns-propagation-seconds 60 --dns-duckdns-token dns_duckdns_token=<my-token>`), I do get the same error stating that the TXT update could not be set.

However, if I follow the wildcard example found on the [certbot-dns-duckdns PyPi page](https://pypi.org/project/certbot-dns-duckdns/), it appears to include an additional argument (`--preferred-challenges dns`). But most notably, when reproducing my steps, I realized that the argument that NPM had populated for `--dns-duckdns-token` was formatted incorrectly:

NPM had sent:
```
--dns-duckdns-token dns_duckdns_token=<my-token>
```
instead of the `certbot-dns-duckdns` example:
```
--dns-duckdns-token <my-token>
```
It appears that NPM had the help-text populated incorrectly and showed the format which is expected to be in a credential file, not the format which is expected by the `--dns-duckdns-token` argument which is called. The credentials file format as shown in the help text would be valid if one were to pass a filepath as the argument to `--dns-duckdns-credentials`. 
![image](https://user-images.githubusercontent.com/17794229/120942006-5381b600-c6fc-11eb-8c46-c8de61d18424.png)
![image](https://user-images.githubusercontent.com/17794229/120942020-72804800-c6fc-11eb-9766-9135866b19e0.png).

When I stripped the `dns_duckdns_token=` prefix from the token I entered in NPM's "Credentials File Content" field, the wildcard cert was generated successfully despite never having been generated manually, as described in the original issue.

I believe the fix for this should be a UI update. The input field should be labeled: "DuckDNS Token" instead of "Credentials File Content", and the pre-populated help text should be `your-duckdns-token` instead of `dns_duckdns_token=your-duckdns-token`. This is the correct format for the `--dns-duckdns-token` argument which this content is passed to.
